game/src/test/scala/io/aigar/game/GameThreadSpec.scala

```diff
@@ -0,0 +1,28 @@
+import io.aigar.game._
+import org.scalatest._
+
+
+class GameThreadSpec extends FlatSpec with Matchers {
+  "A GameThread" should "not have a ranked game state at first" in {
```

Comment:
Maybe it is because before making it a ranked game we want to check if there's already one. If so, we making a private game for the team?